tests/test_association.py

```diff
@@ -135,18 +121,33 @@ def mockGet(datasetType, dataId=None):
                 elif datasetType == 'deepDiff_diaSrc':
                     return testDiaSources
             raise dafPersist.NoResults("Dataset not found:", datasetType, dataId)
-        self.butler = NonCallableMock(spec=dafPersist.Butler, get=mockGet)
 
         self.numTestDiaObjects = 5
         self.diaObjects = createTestPoints(
-            pointLocsDeg=[[idx, idx] for idx in
-                          range(self.numTestDiaObjects)],
-            schema=make_minimal_dia_object_schema(['r']))
+            5, schema=make_dia_object_schema())
         for diaObject in self.diaObjects:
             diaObject['nDiaSources'] = 1
 
+        self.butler = NonCallableMock(spec=dafPersist.Butler, get=mockGet)
```

Comment:
Can you please keep the `NonCallableMock` and `mockGet` definitions together? It's more confusing with the unrelated `self.diaObjects` definition in between.